packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -200,13 +186,29 @@ export class FacetSearch {
     ));
   }
 
+  private get showNoValuesFound() {
+    return (
+      this.component.facetSearchQuery !== '' &&
+      !this.facetSearchResults.length &&
+      !this.facetSearchState.isLoading
+    );
+  }
+
+  private get noValuesFound() {
```

Comment:
adding noValuesFound has a div ouside the ul in order to not make it selectable with arrow keys